lib/session/siprec-call-session.js

```diff
@@ -26,9 +26,9 @@ class SipRecCallSession extends InboundCallSession {
       this.metadata = reMetadata;
 
       if (this.ep && this.ep2) {
-        let remoteSdp = this.sdp1.replace(/sendonly/, 'sendrecv');
+        let remoteSdp = this.sdp1.replace(/sendrecv/, 'sendonly');
```

Comment:
I think this will actually be a problem.  See the most recent code in this file with the comment
```
      /* note: in order to get the two streams to exchange media through freeswitch, we need them to be sendrecv */
      let remoteSdp = this.sdp1.replace(/sendonly/, 'sendrecv');
```

Reply:
but this ack that needs to be sent to source as sendonly , for the freeswitch is done below

https://github.com/vdharashive/jambonz-feature-server/blob/main/lib/session/siprec-call-session.js#L49

Reply:
@davehorton you are right , ill revert this commit